### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by jspahrsummers**
@Autoc0diq @erichoracek Feedback welcome!

**To do:**

 - [x] Unit tests

### CODE REVIEW

File: RACAction/RACAction.h [hunk 1]
```diff
@@ -14,6 +14,32 @@ FOUNDATION_EXPORT double RACActionVersionNumber;
...
+
+/// Replays the latest inner signal sent upon -act_executions.
+///
+/// Different subscriptions to this signal may connect to different executions
+/// of the action.
+///
+/// Returns a signal that, upon subscription, will replay events from the
+/// current execution so far, then forward any future events, or else the
+/// previous execution if the action is not currently executing.
+@property (nonatomic, strong, readonly) RACSignal *act_latestExecution;
```
Comment (erichoracek) [inline RACAction/RACAction.h]: Can this signal error?

Comment (jspahrsummers) [inline RACAction/RACAction.h]: Yeah. I'll clarify that.

**Comment by jspahrsummers** [comment]
Tests added. Ready for review and merge.

**Comment by erichoracek** [comment]
Looks good. :+1:

**Pull request merged**